Packs/Oletools/Scripts/OletoolsScript/OletoolsScript.yml

```diff
@@ -0,0 +1,134 @@
+commonfields:
+  id: Oletools
+  version: -1
+name: Oletools
+script: ''
+type: python
+args:
+- default: false
+  description: The ole command to activate. 
+              oleid - to analyze OLE files to detect specific characteristics usually found in malicious files.
+              oleobj - to extract embedded objects from OLE files.
+              olevba - to extract and analyze VBA Macro source code from MS Office documents (OLE and OpenXML).
+  isArray: false
+  name: ole_command
+  required: true
+  secret: false
+  predefined:
+    - 'oleid'
+    - 'oleobj'
+    - 'olevba'
+- default: false
+  description: The file to activate the oletools analysis on.
+  isArray: false
+  name: entryID
+  required: true
+  secret: false
+- default: false
+  description: Display all the obfuscated strings with their decoded content (Hex, Base64, StrReverse, Dridex, VBA). Note that this works only with the olevba command.
+  isArray: false
+  name: decode
```

Comment:
As we discussed, this should be a list as well (true or false).

Reply:
Done